moveit_core/utils/CMakeLists.txt

```
@@ -5,23 +5,48 @@ add_library(${MOVEIT_LIB_NAME} SHARED
   src/message_checks.cpp
   src/rclcpp_utils.cpp
 )
+<<<<<<< HEAD
 ament_target_dependencies(${MOVEIT_LIB_NAME} Boost moveit_msgs)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+=======
+target_include_directories(moveit_utils PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include/moveit_core>
+)
+ament_target_dependencies(moveit_utils Boost moveit_msgs rclcpp fmt)
+target_link_libraries(moveit_utils rsl::rsl)
+set_target_properties(moveit_utils PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+>>>>>>> 3df6b2813 (Depend on `rsl::rsl` as a non-Ament dependency (#2578))
```

```suggestion
ament_target_dependencies(${MOVEIT_LIB_NAME} Boost moveit_msgs)
set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
```

RSL isn't being used so we can ignore the incoming changes.